ftw/file/browser/tinymce.py

```diff
@@ -0,0 +1,20 @@
+from plone.app.uuid.utils import uuidToObject
+from Products.TinyMCE.adapters.JSONFolderListing import JSONFolderListing
+import simplejson as json
```

Comment:
Could you update the json import here too since we have no ``simplejson`` dependency?